package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gus-markdown",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
The package has the version that we want, I'm not able to change the manifest though because chrome won't let you release a version that is less than a version you have previously released.